core/designsystem/src/main/java/com/moya/funch/icon/FunchIconAsset.kt

```diff
@@ -0,0 +1,22 @@
+package com.moya.funch.icon
+
+import com.moya.funch.designsystem.R
+
+object FunchIconAsset {
+
+    val arrow_back_android_24 = R.drawable.ic_arrow_back_android_24
+    val arrow_right_android_24 = R.drawable.ic_arrow_right_android_24
+    val arrow_left_small_24 = R.drawable.ic_arrow_left_small_24
+    val arrow_right_small_24 = R.drawable.ic_arrow_right_small_24
+    val arrow_up_limit_24 = R.drawable.ic_arrow_up_limit_24
+    val diagonal_24 = R.drawable.ic_diagonal_24
+    val exclamation_circle_red_24 = R.drawable.ic_exclamation_circle_red_24
+    val information_24 = R.drawable.ic_information_24
+    val left_right_24 = R.drawable.ic_left_right_24
+    val minus_24 = R.drawable.ic_minus_24
+    val search_24 = R.drawable.ic_search_24
+    val search_16 = R.drawable.ic_search_16
```

Comment:
```suggestion
object Search {
    val big = R.drawable.ic_search_24
    val small = R.drawable.ic_search_16
}
```

icon이 추가될수록 구분하기 좀 힘들수도 있겠다는 생각이 드는데 이렇게 분기처리도 해보면 어때여?!

Reply:
아이콘 분류에 따라 좀 더 명확하고 쉽게 관리하기 위해서 object로 나누어도 좋아 보이네요! 이거 적용하겠습니다!! 
```
object Search {
    val big = R.drawable.ic_search_24
    val small = R.drawable.ic_search_16
}
```
대신 우선은 기존 네이밍의 형식(`val search_16 = R.drawable.ic_search_16`)은 그대로 진행해 볼게요. 수치를 표시해야 좀 더 명확하게 보일것 같습니다!